pr_check.sh

```diff
@@ -20,12 +20,12 @@ IQE_FILTER_EXPRESSION=""
 
 set -exv
 
-CHROME_SHA=$(curl -X GET "https://quay.io/api/v1/repository/cloudservices/insights-chrome-frontend/tag/?onlyActiveTags=true" | jq .tags\[0\].name -r)
+CHROME_SHA=$(curl -X GET "https://quay.io/api/v1/repository/cloudservices/insights-chrome-frontend/tag/?onlyActiveTags=true&limit=100" | jq '.tags | [.[] | select(.name | test("^[a-zA-Z0-9]{7,40}$"))] | .[0].name' -r)
```

Comment:
We noticed the pipeline was pulling `pr-xxxx` shas of chrome. Once we switch to the konflux `latest` repo/image of chrome this shouldn't be an issue. For now I've added a regex to test the name of the tag (either 7 alphanumeric characters for the short SHA or 40 for the full sha).